Sidney-/slugtrade
# Signup style

Completed signup form with styling. 

Testing: 

1. make sure it looks nice in different browsers
2. check that validation pops up for each input (click submit without a value in each input) and make sure error messages look nice (no mushed to on side or something)
3. upload an image and make sure preview is working
4. submit new user and be redirected to products page
